### ISSUE TITLE
Making this process more better

### ISSUE THREAD

**Issue by ameerthehacker** [electron/electron-apps#531]
I feel like we can have a website which has a dedicated way to add apps and details without making a pull request. This would be quick and fast and we can ad much more features like ratings download counter etc. If anyone is interested in this idea I would like to create that site for electron putting my entire heart and sole ❤️ . Waiting for your reply.

**Comment by ameerthehacker** [comment]
Exactly but how should i get started? This would be a big change to the site and is that ok and doing this in one pull request does not seem right way for me so i would be great help if you can give your thoughts on this.

**Comment by zeke** [comment]
Yeah honestly I feel like this would be a pretty big undertaking for an outside contributor.

@MarshallOfSound, you were working on something related to this. What came of that? Any thoughts on how it should work?

**Comment by MarshallOfSound** [comment]
I still have the code somewhere, I got to the point where I was inferring default values from a repository and was making progress towards automatically making a PR for the users app.  When I get a chance I'll resurrect the code from whatever stash it is sitting in and push it up so others can maybe finish off the work

**Comment by ameerthehacker** [comment]
Great, but will it be a problem if I can start things from scratch and start pushing minor updates step by step. This is the roadmap which I feel might work.

1. Create login with Github
2. Get the default values from repo
3. Store those things in the database 
4. Upload the downloadables to a S3 bucket or something similar
5. Making sure backward compatibility for the older apps and the apps being newly added.

**Comment by MarshallOfSound** [comment]
@ameerthehacker I don't think we want to move away from storing everything in a GitHub repo, it allows for versioning/tracking/history we won't get from an S3 bucket.  Steps 1-3 I already have somewhere with local code and step 4 is what I stopped working on just instead of sending to S3 I was making it auto-PR to github.  I'll figure out which laptop had the code on it some time this week if I get a chance

**Comment by ameerthehacker** [comment]
Fine @MarshallOfSound I will be waiting for it

**Comment by Rajdeep-TG** [comment]
@MarshallOfSound Is there any progress on it? What about a dashboard that will trigger the electron-bot to add the app to the repo (with a direct commit or PR) after an app is reviewed? And after the PR is merged, it'll automatically add that app to the meta files and rebuild the https://www.electronjs.org website. This repo will still be the centralised place for all apps but a Dashboard will actually make the process better and easier - both for reviewers and developers.
